Trappist/src/Promact.Trappist.Core/Controllers/TestController.cs

```diff
@@ -12,29 +12,40 @@ public class TestsController : Controller
     {
         private readonly ITestsRepository _testRepository;
         private readonly IStringConstants _stringConstant;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the "_stringConstant" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1450)